config/__init__.py

```diff
@@ -0,0 +1,14 @@
+from os import environ
+
+FLASK_ENV = environ.get("FLASK_ENV")
+
+if FLASK_ENV == "development":
+    from config.environments.development import DevelopmentConfig as Config
+elif FLASK_ENV == "dev":
+    from config.environments.dev import DevConfig as Config
```

Comment:
are these meant to be two different envs?

Reply:
Yep exactly